index.js

```diff
@@ -14,13 +14,30 @@ module.exports = {
     });
 
     // JQuery UI
-    ['core', 'widget', 'mouse', 'draggable', 'resizable'].forEach(function(module) {
+    app.import({
+      development: app.bowerDirectory + '/jquery-ui/jquery-ui.js',
+      production:  app.bowerDirectory + '/jquery-ui/jquery-ui.min.js'
+    });
+    
+    ['widget', 'plugin'].forEach(function(module) {
       app.import({
         development: app.bowerDirectory + '/jquery-ui/ui/' + module + '.js',
         production:  app.bowerDirectory + '/jquery-ui/ui/minified/' + module + '.min.js'
       });
     });
 
+    ['mouse', 'draggable', 'resizable'].forEach(function(module) {
+      app.import({
+        development: app.bowerDirectory + '/jquery-ui/ui/widgets/' + module + '.js',
+        production:  app.bowerDirectory + '/jquery-ui/ui/widgets/minified/' + module + '.min.js'
+      });
+    });
+    
+    app.import({
+      development: app.bowerDirectory + '/jquery.ui.touch/jquery.ui.touch.js',
```

Comment:
Gridstack has `touch` as an optional dependency:
https://github.com/troolee/gridstack.js#requirements

We don't have to handle it in this PR, but we should consider letting users set a flag in `environment.js` to opt out of including these files. Similar to disabling `ember-cli-mirage`.

Reply:
@tacocat I implemented the most basic support for this in https://github.com/yahoo/ember-gridstack/pull/9/commits/3bc5cae65285745ecb28cd65d3c9020572b5b029 and added to this PR. Please let me know if you want to include it here *or* move to a different PR (where it can be expanded to allow granular exclusion of all bower dependencies).

I think the current implementation is easily expandable to a more generic solution in the future (the API or configuration option would not need to change) so I don't think there's danger in including the current implementation here.